packages/ethereum/ethereum-light-client/Cargo.toml

```diff
@@ -28,3 +28,6 @@ hex = { workspace = true }
 
 # Only used for test-utils feature
 milagro_bls = { workspace = true, optional = true }
+ibc-proto-eureka = { workspace = true, optional = true }
+prost = { workspace = true, optional = true }
+ibc-eureka-solidity-types = { workspace = true, optional = true }
```

Comment:
Is it eventually possible for these to go to a separate crate, such as `ethereum-test-utils`